### PR TITLE
Plugins: add level and signature badges to plugin details page

### LINKED ISSUE #28900: Display plugin signature details
In #28899 we are implementing support for exposing the new plugin signature detail fields via the API.  The details of the plugin signature then need to be exposed in the UI.

@sarlinska has put together mockups for this

### LINKED ISSUE #33552: Add details badges for plugin signing
The plugin details page should show signature and org badges for a signed plugin.

### PR DESCRIPTION
**Pull request by jackw**
**What this PR does / why we need it**:
This PR adds badges to display a signed plugins level and org. It also fixes the icon margins in the sidebar and updates the plugins search input so a user can search by type (e.g. 'datasource' or 'panel').

Screenshot 2021-04-30 at 08 35 20
Screenshot 2021-04-30 at 08 35 29
Screenshot 2021-04-30 at 08 35 51
Screenshot 2021-04-30 at 08 36 04

**Which issue(s) this PR fixes**:

Fixes #33552
Fixes #28900

**Special notes for your reviewer**:
Where should `<PluginSignatureDetailsBadge />` live? In grafana-ui next to `<PluginSignatureBadge />`  or next to the component that uses it?

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Looks good! Awesome job!

File: public/app/features/plugins/PluginPage.tsx [hunk 1]
```diff
@@ -500,4 +523,71 @@ export function loadPlugin(pluginId: string): Promise<GrafanaPlugin> {
...
+
+  return (
+    <>
+      {signatureType && (
+        <Badge
+          color="green"
+          className={styles.badge}
+          text={
+            <>
+              <strong className={styles.strong}>Level:&nbsp;</strong>
```
Comment (dprokop) [inline public/app/features/plugins/PluginPage.tsx:555]: I guess this is a well-thought name, but I personally find Level quite a weird name 😬

**Review by dprokop — APPROVED** [review]
Those cards look awesome. Great work @jackw

File: public/app/features/plugins/PluginPage.tsx (hunk 1 above)
Comment (jackw) [reply public/app/features/plugins/PluginPage.tsx:555]: Yeah, I thought the same plus the code (BE and FE) refers to it as "type".

**Comment by hemdrup** [comment]
I assume this notification will bit sticky/permanent until dismissed, as it’s actionable.
So, it would need an [x] to be dismissed. But, I think all the notification are missing it :slightly_smiling_face: 

Screenshot 2021-05-03 at 10 06 00
Screenshot 2021-05-03 at 10 05 50

**Pull request merged**